src/components/form-builder/designer/designer.tsx

```
@@ -147,19 +147,19 @@ export function Designer() {
                     ref={droppable.setNodeRef}
                 >
                     {!droppable.isOver && elements.length === 0 && (
-                        <p className="flex grow items-center text-3xl font-bold text-muted-foreground">
+                        <p className="text-muted-foreground flex grow items-center text-3xl font-bold">
                             Drop Here
                         </p>
                     )}
                     {droppable.isOver && elements.length === 0 && (
                         <div className="w-full p-4">
-                            <div className="h-[120px] rounded-md bg-primary/20"></div>
+                            <div className="bg-primary/20 h-[120px] rounded-md"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)